daemon/daemon.go

```diff
@@ -980,6 +985,10 @@ func initNetworkController(config *Config) (libnetwork.NetworkController, error)
 	return controller, nil
 }
 
+func (daemon *Daemon) NetworkApiRouter() func(w http.ResponseWriter, req *http.Request) {
```

Comment:
Yes. this (writer and request) is something Gorilla mux expects for handling the APIs. (Please refer to the api/server/server.go).
The reason I decided to keep this part of Daemon is that I didnt feel it is correct to expose the networkController out of daemon. Hence added this helper method which provides the APIHandler for network APIs.